nethserver-directory.spec

```
@@ -44,6 +44,8 @@ rm -rf %{buildroot}
 %ghost %attr(644,root,root) /etc/pam.d/password-auth-nh
 %ghost %attr(644,root,root) /etc/pam.d/system-auth-nh
 %config %attr (0440,root,root) %{_sysconfdir}/sudoers.d/20_nethserver_directory
+%attr (0750,root,root) /usr/libexec/nethserver/slapd-create-dhparam
+%config(noreplace) %attr (0440,root,ldap) /etc/openldap/certs/slapd.dh.params
```

I'd really like a standard file extension. As the file is in PEM format, the extension can be .pem: what else?

```suggestion
%config(noreplace) %attr (0440,root,ldap) /etc/openldap/certs/slapd-dhparam.pem
```